explorer/src/constants/columns/portal.tsx

```diff
@@ -55,3 +56,29 @@ export const columns = ({ chain }: { chain: Chain }): ColumnDef<Portal>[] => [
     },
   },
 ];
+
+export const portalColumnsOption = {
+  name: {
+    width: "25%",
+  },
+  description: {
+    width: "50%",
+  },
+  owner: {
+    width: "25%",
+  },
+};
```

Comment:
I don't think this works.
In dev mode, go to `http://localhost:5173/linea/modules/0x160f0ddc9aec86b7d5618230ad49b446f97acd57` => you get an error

Reply:
done